src/System.Drawing.Common/tests/Imaging/EncoderParameterTests.cs

```diff
@@ -318,7 +318,7 @@ public void Ctor_Encoder_TooBigNumberOfValues_Type_Value_AccessViolationExceptio
         [InlineData(int.MinValue)]
         public void Ctor_Encoder_NegativeNumberOfValues_Type_Value_OutOfMemoryException(int numberOfValues)
         {
-            if (numberOfValues == -1 && PlatformDetection.IsUbuntu1710) // [ActiveIssue(24274)]
+            if (numberOfValues == -1 && (PlatformDetection.IsUbuntu1710 || PlatformDetection.IsUbuntu1804)) // [ActiveIssue(24274)]
```

Comment:
You can leave the new property as it may be useful but in this place can you use a property like IsUbuntu1710OrHigher so we don't have to keep updating it